db/migrate/20180914010751_create_offerings.rb

```diff
@@ -0,0 +1,9 @@
+class CreateOfferings < ActiveRecord::Migration[5.1]
+  def change
+    create_table :offerings do |t|
+      t.float :amount
```

Comment:
the float is going to fuck you here on the math.  Always use decimal for $

Reply:
added a change_column migration